### PR TITLE
Util v7.1.2 and Tx v3.3.2 Releases

### PR DESCRIPTION
**Pull request by holgerd77**

### CODE REVIEW

File: packages/tx/CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 (modification: no type change headlines) and this project adheres to
 [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## 3.3.2 - 2021-09-30
+
+This release updates the `ethereumjs-util` library to `v7.1.2` which provides a safer conversion of integer values with the `intToHex` and `intToBuffer` functions by replacing the re-exported functions with own implementations which throw on wrong integer input (decimal values, non-safe integers, negative numbers,...), see PR [#1500](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1500).
```
Comment (gabrocheleau) [inline packages/tx/CHANGELOG.md:11]:
```suggestion
This release updates the `ethereumjs-util` library to `v7.1.2` which provides a safer conversion of integer values with the `intToHex` and `intToBuffer` functions by replacing the re-exported functions from `ethjs-utils` with own implementations which throw on wrong integer input (decimal values, non-safe integers, negative numbers,...), see PR [#1500](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1500).
```

**Review by gabrocheleau — APPROVED** [review]
Added a small clarification in the tx/changelog

**Review by ryanio — APPROVED** [review]
lgtm

**Comment by holgerd77** [comment]
Just published the following releases on npm:

- `ethereumjs-util` [v7.1.2](https://github.com/ethereumjs/ethereumjs-monorepo/releases/tag/ethereumjs-util%407.1.2)
- `@ethereumjs/tx` [v3.3.2](https://github.com/ethereumjs/ethereumjs-monorepo/releases/tag/%40ethereumjs%2Ftx%403.3.2)

**Pull request merged**